templates/design_system/authorize_v2.html

```diff
@@ -82,7 +82,7 @@ <h3>Error: {{ error.error }}</h3>
 	</div>
 
 	<div class="bb-c-card ds-u-margin-y--4 ds-u-padding--4 ds-u-margin-x--2 bene-auth-max-width">
-		You can revoke this app’s access to your data at any time by logging into your <a href="https://www.mymedicare.gov/">MyMedicare.gov</a> account or calling us at 1-800-633-427.
+		You can revoke this app’s access to your data at any time by logging into your <a href="https://www.mymedicare.gov/">Medicare.gov</a> account or calling us at 1-800-633-427.
```

Comment:
Completely unrelated to your changes but there appears to be a typo in the phone number.

Reply:
Nice catch. I'll push a change.